doc/inspection.rst

```diff
@@ -17,6 +17,7 @@ to diagnose issues with model performance.
 
 .. rubric:: Examples
 
+* :ref:`sphx_glr_auto_examples_inspection_plot_causal_interpretation.py`
```

Comment:
this example doesn't have a single inspection import, so removing it from here. That example certainly deserves to be linked from more places, but I'm not sure where.